tests/ui/README.md

```diff
@@ -25,7 +25,7 @@ The tests must be run in Firefox 48 or later.
 To run against a different environment, pass in a value for `--base-url`, like so:
 
 ```bash
-$ tox -e discopane-ui-tests -- --base-url https://discovery.addons.mozilla.org
+PYTEST_ADDOPTS="--base-url https://discovery.addons.mozilla.org" tox -e discopane-ui-tests
```

Comment:
The driver argument should be picked up from https://github.com/mozilla/addons-frontend/blob/master/tests/ui/setup.cfg#L2 so this shouldn't be necessary. For what it's worth, f you update to the latest version of pytest-base-url you can just add an environment variable of `PYTEST_BASE_URL` with the value of the URL. Same goes for Travis.